### ISSUE TITLE
Gradient Backward error

### ISSUE THREAD

**Issue by morgan-bc** [intel/intel-extension-for-pytorch#193]
Enviroment:
- System: Ubuntun-20.04
- gcc: 9.3.0
- python: 3.8
- python: 1.9.1+cpu
- ipex: 1.9.0

When I use ipex.DEVICE intead of cpu, the model throw errors:
```
RuntimeError: Function MulBackward0 returned an invalid gradient at index 0 - expected type TensorOptions(dtype=float, device=cpu, layout=Strided, requires_grad=false (default), pinned_memory=false (default), memory_format=(nullopt)) but got TensorOptions(dtype=float, device=xpu:0, layout=Strided, requires_grad=false (default), pinned_memory=false (default), memory_format=(nullopt))

```
For other simple models like classification resnet, moboile_net, the `ipex.DEVICE` works well. I tested classification mobile_net_v3 and found that the acceleration effect is not obvious and the training computation event become slower. 

In my personal experice, the ipex either can not run or can not achieve the optimization effect. Dose the code need to be carefully modified to match ipex ?

**Comment by EikanWang** [comment]
It is a known issue for training and will be fixed at 1.10.

**Comment by EikanWang** [comment]
This issue has been fixed on release/1.10 branch. @morgan-bc , please feel free to reopen this issue if you still encounter this issue.
